src/test/java/io/aiven/kafka/connect/s3/testutils/BucketAccessor.java

```diff
@@ -50,6 +54,29 @@ public final void createBucket() {
         s3.createBucket(bucketName);
     }
 
+    public final void removeBucket() {
+        final var chunk = s3.listObjects(bucketName)
+                                          .getObjectSummaries()
+                                          .stream()
+                                          .map(S3ObjectSummary::getKey)
+                                          .toArray(String[]::new);
+
+        final var deleteObjectsRequest =
+            new DeleteObjectsRequest(bucketName).withKeys(chunk);
+        try {
+            s3.deleteObjects(deleteObjectsRequest);
+        } catch (final MultiObjectDeleteException e) {
+            for (final var err : e.getErrors()) {
+                System.out.println(String.format("Couldn't delete object: %s. Reason: [%s] %s",
```

Comment:
better logging here ;-)

Reply:
but its a for tests only

Reply:
I agree with Andrey, why not use logging in tests as well. We need the test-level logging config though, like this one https://github.com/aiven/aiven-kafka-connect-gcs/blob/master/src/test/resources/logback-test.xml

Reply:
Ok. Lets use logging

Reply:
For this not to sound cargo: the primary rationale for blending the logging from the code-under-test with the test logging together is for them to end up in a single destination, properly ordered despite the way the CI environment and/or Gradle collect the output. Sometimes they do non-trivial output forwarding when e.g. running tests in parallel, etc. This connector is a simple project with relatively simple test suite and the logging settings, so this might not be very articulated, but I'm advocating the general good practice here.

Reply:
Agree with Ivan, we can loose error messages due to diff configuartion of `stdout` and `stderr` .